protos/group.proto

```diff
@@ -0,0 +1,13 @@
+syntax = "proto3";
+
+import "common.proto";
+
+package dcs;
+
+message GetUnitsRequest {
+  string group_name = 2;
```

Comment:
I am not sure about the inconsistency between `GetUnitsRequest` and `GetUnitPositionRequest `, where this one here uses `group_name` while the latter uses `name`. I wen with this for now, because the latter has `Unit` already in its method name.

Reply:
Would it be clearer to name this `GetUnitsInGroup` which would make it clearer why this is `group_name` but the position would be `name` (since it is more obviously for a specific unit).

Reply:
Would it be clearer to name this GetUnitsInGroup which would make it clearer why this is group_name but the position would be name (since it is more obviously for a specific unit).

Reply:
I like the suggestion of using namespaces here. I've moved it into a namespace in here https://github.com/DCS-gRPC/rust-server/pull/11/commits/81a5bd7ddda73731d17e712d56e6f7efae15782c, to see what it would look like. I like it, wdyt? Though it probably means that we should also add namespaces to all the other types for consistency reasons later on.

Reply:
Adding everything into namspaces is part of the refactoring I want to do once this PR is merged.